portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -41,19 +47,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
-  /* 
-  * when the submit button on the values page is hit, 
-  * doPost requests the text in the author and commment field and stores them
-  */ 
-  @Override
+    @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    this.username = request.getParameter("username");
+    this.reflection = request.getParameter("reflection");
 
-    // Get the input from the comments form
-    String name = request.getParameter("user");
-    String reflection = request.getParameter("reflection");
-    comments.add(new Comment(name, reflection));
+    Entity taskEntity = new Entity("comment");
+    taskEntity.setProperty("username", username);
+    taskEntity.setProperty("reflection", reflection);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
```

Comment:
You could have a seperate helper method for putting entities into the datastore.

Reply:
+1